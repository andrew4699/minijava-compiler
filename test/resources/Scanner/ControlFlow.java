while(x < 5) {
  return 2;
}
