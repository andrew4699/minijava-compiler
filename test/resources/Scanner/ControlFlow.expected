WHILE LPAREN ID(x) LESSTHAN INT(5) RPAREN LBRACE
  RETURN INT(2) SEMICOLON
RBRACE
